batch/sc-batch-fargate.yaml

```diff
@@ -0,0 +1,271 @@
+AWSTemplateFormatVersion: "2010-09-09"
+Description: 'AWS Batch with Fargate'
+Metadata:
+  cfn-lint:
+    config:
+      ignore_checks:  # Ignore cfn-lint errors for PyPlate.
+        - W2001
+  AWS::CloudFormation::Interface:
+    ParameterGroups:
+      - Label:
+          default: Container Image
+        Parameters:
+          - Image
+          - Memory
+          - Command
+      - Label:
+          default: Environment
+        Parameters:
+          - Secrets
+          - EnvVars
+      - Label:
+          default: Schedule Job
+        Parameters:
+          - EnableSchedule
+          - Schedule
+Parameters:
+  Image:
+    Type: String
+    Description: >
+      The container image to run, must be from a public dockerhub or AWS ECR repository.
+    Default: "debian:latest"
+    ConstraintDescription: >
+      Must be a valid dockerhub or AWS ECR container image
+      Example: debian:latest or 111111111.dkr.ecr.us-east-1.amazonaws.com/MY-IMAGE:latest
+  Memory:
+    Description: >
+      The amount (in MiB) of memory for the container. Mapping of memory to CpuShares for the
+      container is 1024: 0.25, 2048: 0.5, 4096: 1, 10240: 2, 18432: 4
+    Type: String
+    Default: "1024"
+    AllowedValues:
+      - "1024"
+      - "2048"
+      - "4096"
+      - "10240"
+      - "18432"
+  EnableSchedule:
+    Description: >
+      true to run on a schedule, false to disable. If enabled a valid Schedule must be provided
+    Type: String
+    Default: false
```

Comment:
Oh interesting, this just run single process jobs too?  What does that look like as a SC product?